stl/inc/fstream

```diff
@@ -48,6 +48,18 @@ namespace experimental {
 } // namespace experimental
 #endif // _FSTREAM_SUPPORTS_EXPERIMENTAL_FILESYSTEM
 
+// clang-format off
+template <class _Ty>
+_INLINE_VAR constexpr bool _Is_any_path = _Is_any_of_v<_Ty
+#if _FSTREAM_SUPPORTS_EXPERIMENTAL_FILESYSTEM
+    , experimental::filesystem::path
+#endif // _FSTREAM_SUPPORTS_EXPERIMENTAL_FILESYSTEM
+#if _HAS_CXX17
+    , filesystem::path
+#endif // _HAS_CXX17
+    >;

```

Comment:
so, `_Is_any_of_v` does work (and is always false) if there's nothing but `_Ty` in it's parameter list right?

Reply:
Yip: https://godbolt.org/z/7Evvn4MEz.